<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="7" total="8" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="7935" started-at="2018-02-15T10:52:54Z" finished-at="2018-02-15T10:53:02Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="7935" started-at="2018-02-15T10:52:54Z" finished-at="2018-02-15T10:53:02Z">
      <class name="Test_Cases.Verify_dashboard">
        <test-method status="PASS" signature="Openbrowser()[pri:0, instance:Test_Cases.Verify_dashboard@68034211]" name="Openbrowser" is-config="true" duration-ms="8382" started-at="2018-02-15T16:22:46Z" finished-at="2018-02-15T16:22:54Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Openbrowser -->
        <test-method status="FAIL" signature="verify_resto()[pri:0, instance:Test_Cases.Verify_dashboard@68034211]" name="verify_resto" duration-ms="548" started-at="2018-02-15T16:22:54Z" finished-at="2018-02-15T16:22:55Z">
          <exception class="javax.imageio.IIOException">
            <message>
              <![CDATA[Can't read input file!]]>
            </message>
            <full-stacktrace>
              <![CDATA[javax.imageio.IIOException: Can't read input file!
	at javax.imageio.ImageIO.read(ImageIO.java:1301)
	at lib_methods.Comapre_images.compare_hideen_image(Comapre_images.java:88)
	at com.pageObject.Home.verify_restautants(Home.java:386)
	at Test_Cases.Verify_dashboard.verify_resto(Verify_dashboard.java:32)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- javax.imageio.IIOException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verify_resto -->
        <test-method status="PASS" signature="tracerror(org.testng.ITestResult)[pri:0, instance:Test_Cases.Verify_dashboard@68034211]" name="tracerror" is-config="true" duration-ms="333" started-at="2018-02-15T16:22:55Z" finished-at="2018-02-15T16:22:55Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=verify_resto status=FAILURE method=Verify_dashboard.verify_resto()[pri:0, instance:Test_Cases.Verify_dashboard@68034211] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tracerror -->
        <test-method status="PASS" signature="teardown()[pri:0, instance:Test_Cases.Verify_dashboard@68034211]" name="teardown" is-config="true" duration-ms="7031" started-at="2018-02-15T16:22:55Z" finished-at="2018-02-15T16:23:02Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- Test_Cases.Verify_dashboard -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
